assets/javascripts/discourse/templates/components/channel-list.hbs

```diff
@@ -42,6 +42,7 @@
       {{#each publicChannels as |channel|}}
         {{chat-channel-row channel=channel switchChannel=onSelect}}
       {{/each}}
+      {{plugin-outlet name="below-plugin-chat-channels" tagName="" args=(hash inSidebar=inSidebar)}}
```

Comment:
Should that outlet appear only when there are channels? (i.e. should it be inside that `if` block or below it?)

Reply:
Good question.

Reply:
IDK

Reply:
Depends on the use-case, I guess. Probably would be better outside the `if`.